setup.py

```diff
@@ -68,7 +68,7 @@
     'idutils>=1.1.7',
     'invenio-assets>=1.2.2,<1.3.0',
     'invenio-communities>=2.1.1,<3.0.0',
-    'invenio-drafts-resources>=0.6.2,<0.7.0',
+    'invenio-drafts-resources>=0.7.0,<0.8.0',
     'invenio-formatter[badges]>=1.1.0a1,<2.0.0',
     'invenio-i18n>=1.3.0a1',
     'invenio-records>=1.4.0a4,<2.0.0',
```

Comment:
Minor: can you also bump ``invenio-previewer>=1.3.0,<1.4.0'``

Reply:
Done.